### PR TITLE
Prepare v1.12 stable branch

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

File: .github/workflows/build-images-releases.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ name: Image Release Build
 on:
   push:
     tags:
-      - v[0-9]+.[0-9]+.[0-9]+
-      - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+      - v1.12.[0-9]+
+      - v1.12.[0-9]+-rc[0-9]+
```
Comment (joestringer) [inline .github/workflows/build-images-releases.yaml:7]: I'm always a bit confused which of these should be configured on master vs. the older branch. Doesn't the master branch version of this workflow already cover the 1.12 releases?

File: install/kubernetes/Makefile.values [hunk 2]
```diff
@@ -6,18 +6,10 @@ include $(DIGESTS_PATH)
...
-export CILIUM_REPO:=quay.io/cilium/cilium-ci
-export CLUSTERMESH_APISERVER_REPO:=quay.io/cilium/clustermesh-apiserver-ci
-
-# When branching, delete the following lines to defer to the branch's versions
-export CILIUM_VERSION:=latest
-export CILIUM_OPERATOR_SUFFIX=-ci
+export PULL_POLICY:=IfNotPresent
+export CILIUM_BRANCH:=v1.12
+export CILIUM_REPO:=quay.io/cilium/cilium
+export CLUSTERMESH_APISERVER_REPO:=quay.io/cilium/clustermesh-apiserver
```
Comment (joestringer) [inline install/kubernetes/Makefile.values:12]: :+1:

**Review by joestringer — APPROVED** [review]
LGTM. I think I get it, any workflows which run automatically on PRs must be updated to point to 1.12. Any workflows triggered by comment are configured on the master branch and therefore must be removed here. Only one I wasn't sure about was the release tags image build trigger (comment below).

File: .github/workflows/build-images-releases.yaml (hunk 1 above)
Comment (aanm) [reply .github/workflows/build-images-releases.yaml:7]: I got confused about this as well but then I remembered that this GH workflow is executed for tags pushed **into** this branch. That's why we have a specific GH workflow for each branch.

**Pull request merged**